docs/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+
+## 1.0.0b1
+
+Feature:
```

Comment:
I believe the main feature of this release is that load_file, by default, uses native approaches to load datasets into BQ and Snowflake. The old behaviour can be re-enabled by setting `use_native_support=False`. And we should refer users to adequate documentation, so they can become familiar with the new arguments available in `load_file`, including `native_support_kwargs` and `enable_native_fallback`.

Reply:
A second feature can be the dynamic task mapping example

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/576/commits/88b0e130c53839808f7fa7be3df528526f57e8ab)